frontend/components/admin/distributionConfig.vue

```diff
@@ -1,33 +1,249 @@
+          <v-icon>add</v-icon>
+        </v-btn>
+      </template>
+      <v-card>
+        <v-toolbar flat color="white">
+          <v-toolbar-title>Distribution Configs</v-toolbar-title>
+        </v-toolbar>
+        <v-divider></v-divider>
+        <v-card-text>
+          <v-form ref="form" v-model="valid" v-on:submit.prevent="$emit('submit')">
+            <v-layout wrap class="ma-2">
+              <v-flex xs12 md12>
+                <v-select
+                  v-model="dialogValue.lockedWallet"
+                  :label="`Locked Wallet ( ${dialogValue.lockedWallet || 'Address'} )`"
+                  :items="distributionConfigs"
+                  item-value="lockedWallet"
+                  item-text="name"
+                  :clearable="mode==='EDIT' ? false :true"
+                  :disabled="mode==='EDIT' ? true :false"
+                  :rules="fieldRules"
+                ></v-select>
+              </v-flex>
+              <v-flex xs12 md12>
+                <v-text-field v-model="dialogValue.unlockWallet" label="Unlocked Wallet" :rules="fieldRules" :disabled="mode==='EDIT' ? true :false" required></v-text-field>
+              </v-flex>
+              <v-flex xs12 md12>
+                <span>
+                  <strong>Ratio</strong>
+                  ( {{percentage}}% )
+                </span>
+              </v-flex>
+              <v-flex xs6 md6 pa-1 shrink>
+                <v-text-field v-model="dialogValue.ratioDTV" type="number" :rules="fieldRules" label="Ratio DTV"></v-text-field>
+              </v-flex>
+              <v-flex xs6 md6 pa-1 shrink>
+                <v-text-field v-model="dialogValue.ratioDecimals" :rules="fieldRules" type="number" label="Ratio Decimals"></v-text-field>
+              </v-flex>
+            </v-layout>
+            <v-layout wrap justify-end fill-height>
+              <v-btn color="red darken-1" flat @click="clearData">Cancle</v-btn>
+              <v-btn color="green darken-1" flat type="submit" @click="handleSubmit(mode)">Submit</v-btn>
+            </v-layout>
+          </v-form>
+        </v-card-text>
+      </v-card>
+    </v-dialog>
+
+    <v-dialog v-model="ethLoading" max-width="400px">
+      <v-card>
+        <v-card-title>Loading...</v-card-title>
+        <v-progress-linear indeterminate height="2" class="ma-0" style="position: absolute; bottom: 50px; left: 0; right: 0;"></v-progress-linear>
+        <v-card-text>
+          Transection :
+          <a :href="`https://etherscan.io/tx/${transectionHash}`" target="_blank">{{transectionHash}}</a>
+        </v-card-text>
+      </v-card>
+    </v-dialog>
+
+    <v-dialog v-model="confirmDelete" max-width="550px">
+      <v-card>
+        <v-card-title class="headline">Confirm delete</v-card-title>
+        <v-card-text>
+          <p>
+            Delete the distribution config
+            <strong>{{dialogValue.name}}</strong> account?
+          </p>
+          <p>
+            Locked Wallet :
```

Comment:
there is not space before ":" in english. please update the other ones
```suggestion
            Locked Wallet:
```